Content.Client/UserInterface/Systems/Chat/Widgets/ChatBox.xaml.cs

```diff
@@ -184,5 +273,21 @@ protected override void Dispose(bool disposing)
         ChatInput.Input.OnKeyBindDown -= OnInputKeyBindDown;
         ChatInput.Input.OnTextChanged -= OnTextChanged;
         ChatInput.ChannelSelector.OnChannelSelect -= OnChannelSelect;
+        _cfg.UnsubValueChanged(EECVars.ChatStackLastLines, UpdateChatStack); // EE - Chat stacking
+    }
+
+    // EE - Chat stacking
+    private class ChatStackData
```

Comment:
`Class must be explicitly marked as [Virtual], abstract, static or sealed`

```suggestion
    private sealed class ChatStackData
```